src/components/formio/datamap.tsx

```diff
@@ -30,6 +32,13 @@ export const DataMap: React.FC<DataMapProps> = ({name, valueComponent, keyLabel
               const newValue = {...value, [item.key]: event.target.value};
               setValue(newValue);
             },
+            'aria-label': intl.formatMessage(
+              {
+                description: 'Accessible label for error message input field',
+                defaultMessage: 'Error message for "{field}"',
+              },
+              {field: item.key}
```

Comment:
@sergei-maertens I'm not sure if it's possible to define the formatted message outside the `DataMap` component and then format it again with the values from the loop?

Reply:
yeah you can do:

```tsx
// module level constant
const labelMsg = intl.defineMessage({
    description: '...',
    defaultMessage: 'Error message for "{field}"',
});

// ... rest of code

// in loop
intl.formatMessage(labelMsg, {field: item.key});
```